### PR TITLE
Adds .pbf format to support Mapbox Vector Tiles format

### PR DESCRIPTION
**Pull request by luizvital**
Adapted from the work of https://github.com/tilezen/TileStache/blob/integration-1/TileStache/Goodies/VecTiles/mvt.py

### CODE REVIEW

**Comment by alculquicondor** [comment]
Hello @luizvital thanks for the contribution. I'm still going through the code. In the meantime, I would appreciate if you write a test for this new feature.

**Comment by luizvital** [comment]
Hello @alculquicondor. I've just added some tests based on the previous ones.

I had to create some auxiliary functions (coordinate transforms, mock mercantile, convertion to shapely geometries) and reduce precision on the geometry checks, maybe due to the nature of the format conversion.

**Comment by luizvital** [comment]
The padding parameter came to solve this problem: http://stackoverflow.com/questions/39884223/style-vector-tile-layers-point-features-in-openlayers-3-get-cropped

The initial implementation used a hardcoded value of 0. https://github.com/TileStache/TileStache/pull/264/files#diff-a05bca09362224e2ef5fc9a994a40d79R10

File: TileStache/Goodies/VecTiles/pbf.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import mapbox_vector_tile
+
+# extracted from Mapzen TileStache fork version 0.9.0, more recent versions doesn't seem to work.
+# https://github.com/tilezen/TileStache/blob/089ee60f9902e85678499fe8455580a8a013be89/TileStache/Goodies/VecTiles/mvt.py
+
+# coordindates are scaled to this range within tile
+extents = 4096
+
+# tiles are padded by this number of pixels for the current zoom level 
+padding = 0
```
Comment (alculquicondor) [inline TileStache/Goodies/VecTiles/pbf.py]: can we get rid of this?

**Comment by luizvital** [comment]
Thanks for the review @alculquicondor, just made the changes you've requested.

**Comment by alculquicondor** [comment]
Thanks @luizvital

**Comment by luizvital** [comment]
Updated [awesome-vector-tiles](https://github.com/mapbox/awesome-vector-tiles)
https://github.com/mapbox/awesome-vector-tiles/commit/aba35ce17fe1597ecff78aad0fdf3d7e06d586af

**Pull request merged**